GODLIKE.m

```diff
@@ -54,7 +54,9 @@
 %
 %   popsize     positive integer. Indicates the TOTAL population 
 %               size, that is, the number of individuals of all 
-%               populations combined. 

```

Comment:
I'm not so sure about this feature; let me think a bit more on it. 

Do you have some use cases/examples where this actually improves the performance/reliability?